h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -890,10 +890,12 @@ public static int numInts(String... a) {
     return cnt;
   }
 
-  public static boolean isInt(String s) {
-    if (s == null || s.isEmpty()) return false;
-    int i = s.charAt(0)=='-' ? 1 : 0;
-    for(; i<s.length();i++) if (!Character.isDigit(s.charAt(i))) return false;
+  public static boolean isInt(String... ary) {
+    for(String s:ary) {
+      if (s == null || s.isEmpty()) return false;
+      int i = s.charAt(0) == '-' ? 1 : 0;
+      for (; i < s.length(); i++) if (!Character.isDigit(s.charAt(i))) return false;
```

Comment:
There was a bug in the R SVMLight serializer that caused the column indices to be written in sci notation. That reminded me of it.